No-Country-simulation/c19-06-ft-node-react
# This PR sets up the initial Express server for the project.

**Changes:**
- Installed Express and created the initial server setup in `backend/src/index.js`.

**To Test:**
1. Pull the `develop` branch.
2. Run `npm install` to install the new dependencies.
3. Start the server with `npm run dev` and ensure it runs without errors.
4. Navigate to `http://localhost:3001` to see the placeholder response.
